suzieq/poller/worker/services/service.py

```diff
@@ -748,6 +758,32 @@ async def run(self):
             if output:
                 ostatus = [x.get('status', -1) for x in output]
 
+                hostname = output[0]["hostname"]
+                namespace = output[0]["namespace"]
+                key = f'{namespace}-{hostname}'
+
+                should_commit = True
+
+                # we should always commit during the first run of the service
+                # for each node regardless their status.
+                if not self.is_first_run(key):
+                    # If it's not the first run and some command did fail,
+                    # increment the count to be used in hysteresis.
+                    # The info will be committed in a future run.
+                    if any((not Service.is_status_ok(x) for x in ostatus)):
+                        self._consecutive_failures[key] = \
+                            self._consecutive_failures.get(key, 0) + 1
```

Comment:
Nit: we might use a `deafultdict()` with default value to 0 for `self._consecutive_failures`, this allow us to reduce this line to:
```
self._consecutive_failures[key] += 1
```
since when you refer to a not existing key, it automatically creates it assigning the default value (0 in this case).
Even line 780 would benefit from this modification.